expyfun/visual/_visual.py

```diff
@@ -168,7 +168,9 @@ def _check_log(obj, func):
     ptr = cast(pointer(log), POINTER(c_char))
     func(obj, 4096, pointer(c_int()), ptr)
     message = log.value
-    if message:
+    if message.startswith(b'No errors'):
+        pass
+    elif message:
```

Comment:
...and actually, to be backward compatible you almost certainly need to do `if message and not message.startswith(b'No errors'):`